docs/quickstarts/insights-inventory-workspace-rbac/insights-inventory-workspace-rbac.yaml

```diff
@@ -44,28 +44,28 @@ spec:
-            - If you want to give a user full access to ungrouped systems → select all 4 permissions (ungrouped systems are considered a group).
+            - If you want to give a user full access to the workspace and its systems → select all 4 permissions.
+            - If you want to give a user full access to the systems inside a workspace → but not workspace editing access, give them inventory:hosts:read, inventory:hosts:write, and inventory:workspace:read, but NOT inventory:workspace:write.
+            - If you want to give a user full access to ungrouped systems → select all 4 permissions (ungrouped systems are considered a workspace).
         1. Click **Next**.
-        1. On Step 3 (Define inventory group access), for each permission, select groups you would like to apply the permissions to using the dropdown menus. <br>
+        1. On Step 3 (Define inventory workspace access), for each permission, select workspaces you would like to apply the permissions to using the dropdown menus. <br>
         
             Possible scenarios include:
-            - You would like to give users the ability to only see systems in specific groups and NOT ungrouped systems → Select those groups only.
-            - You would like to give users the ability to see systems in specific groups AND any ungrouped systems → Select those groups for all permissions AND “ungrouped systems" for inventory:hosts permissions.
+            - You would like to give users the ability to only see systems in specific workspaces and NOT ungrouped systems → Select those workspaces only.
+            - You would like to give users the ability to see systems in specific workspaces AND any ungrouped systems → Select those workspaces for all permissions AND “ungrouped systems" for inventory:hosts permissions.
             - You would like to give users the ability to see everything in the inventory → There is no need to create a custom role then. You can cancel the creation flow.
         1. Once you’ve made your selections, click **Next**.
         1. Review your selections. Click **Submit**.
 
-        Repeat this process for every Inventory group or groups of users you want to give specific group access to.
+        Repeat this process for every Inventory workspace or workspaces of users you want to give specific workspace access to.
         
-        **NOTE:** If you want to give a group of system administrators all the same access to groups A, B, and C, you can create a single custom role and assign permissions to those 3 groups.
-        However, if you want to give different users access to different groups, you will need to create multiple custom roles - one for each group you would like to give access to.
+        **NOTE:** If you want to give a workspace of system administrators all the same access to workspaces A, B, and C, you can create a single custom role and assign permissions to those 3 workspaces.
```

Comment:
if you want to give a GROUP of system administrators, not a workspace of system administrators hahaha